### DISCUSSION TITLE
Merge these IP TV stations

### DISCUSSION THREAD

**Discussion by hackwin** [iptv-org/community#1088] [Q&A]
Hi, I think you could merge these stations.  They have many of the popular TV brands.  Some have the EPG data, which makes it possible to set up a DVR process.

https://i.mjh.nz/

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-add-a-new-stream-link-to-a-playlists
